### PR TITLE
[Gssoc'21] Added a dropdown menu

### PR DESCRIPTION
**Pull request by arpit456jain**
## Related Issuse

we have to go to home page for going for any check after this we can go to any page .

Closes: #144 
#### Describe the changes you've made

https://user-images.githubusercontent.com/55352601/113331793-fb19eb00-933d-11eb-945c-b0be8016d837.mp4

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Screenshots

|        Original         |          Updated           |
| :---------------------: | :------------------------: |
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

**Comment by arpit456jain** [comment]
@kritikaparmar-programmer I had added the drop down and some hover effect in both mode (dark+light) for easy access.

File: templates/layout.html [hunk 1]
```diff
@@ -38,6 +38,18 @@
               <span class="sr-only">(current)</span>
             </a>
           </li>
+           <li class="nav-item dropdown">
+        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Predictions
```
Comment (kritikaparmar-programmer) [inline templates/layout.html:43]: Rename Predictions to Diseases.

**Review by kritikaparmar-programmer — CHANGES_REQUESTED** [review]
Rest it fine.

File: templates/layout.html (hunk 1 above)
Comment (arpit456jain) [reply templates/layout.html:43]: ok just 1 min

**Comment by arpit456jain** [comment]
@kritikaparmar-programmer kindly add labels before merging

**Review by kritikaparmar-programmer — APPROVED** [review]
Great 👍

**Comment by arpit456jain** [comment]
@kritikaparmar-programmer kindly add labels of GSSoC'21 and level as i didn't get any points

**Comment by arpit456jain** [comment]
@kritikaparmar-programmer  thanks

**Pull request merged**